### PR TITLE
Add AwardWallet

### PR DESCRIPTION
**Pull request by ohheyrj**
Awardwallet allows users to store and track loyalty schemes in one place. Mostly used for air miles but can also deal with store loyalty schemes and hotels. I've included a image and documentation from their FAQ regarding 2FA. Let me know if I have done anything wrong as this is my first time contributing :)

### CODE REVIEW

**Comment by ohheyrj** [comment]
One more thing...
Last time I did a PR I was asked if it fitted the site criteria. I would say it does for the following reasons:

- **Personal Info**: This site allows users to store their account numbers and passwords so that the information can pulled into one place. If this information was accessed it would be possible for someone to either spend air miles/hotel points or in the case of hotel reservations (which AwardWallet also pulls in) cancel or amend bookings.
- **Money**: You can also add in credit card point schemes e.g. American Express Rewards. This information could be used to access a persons account.

**Comment by stephengroat** [comment]
previously looked at in #1956

File: _data/other.yml [hunk 1]
```diff
@@ -49,6 +49,13 @@ websites:
       software: Yes
       doc: https://www.youtube.com/watch?v=5OLcvjVhhuI
 
+    - name: AwardWallet
+      url: https://www.awardwallet.com
```
Comment (jamcat22) [inline _data/other.yml]: This URL redirects to one without a preceding `www`. Please update the url to `https://awardwallet.com`

**Review by jamcat22 — CHANGES_REQUESTED** [review]

**Comment by ohheyrj** [comment]
Hello, I've now made the requested changes.

**Review by jamcat22 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**